src/yang/ietf-tcp.yang

```diff
@@ -209,80 +209,89 @@ module ietf-tcp {
     description
       "Standard TCP congestion control referred to as
        'Reno algorithm'";
-    reference "RFC 5681: TCP Congestion Control";
+    reference
+      "RFC 5681: TCP Congestion Control";
   }
 
   identity new-reno {
     base congestion-control-algorithm;
     description
       "NewReno modification to TCP's fast recovery algorithm";
-    reference "RFC 6582: The NewReno Modification to TCP's
-       Fast Recovery Algorithm";
+    reference
+      "RFC 6582: The NewReno Modification to TCP's Fast Recovery
+       Algorithm";
   }
 
   identity ledbat {
     base congestion-control-algorithm;
     description
       "Low Extra Delay Background Transport (LEDBAT) 
-       congestion control"
-   reference "RFC 6817: Low Extra Delay Background Transport
-       (LEDBAT)";
+       congestion control";
+   reference
+      "RFC 6817: Low Extra Delay Background Transport (LEDBAT)";
   }
 
   identity dctcp {
     base congestion-control-algorithm;
     description
       "TCP Congestion Control for Data Centers (DCTCP)
        congestion control";
-    reference "RFC 8257: Data Center TCP (DCTCP): TCP Congestion
+    reference
+      "RFC 8257: Data Center TCP (DCTCP): TCP Congestion
        Control for Data Centers";
   }
 
   identity cubic {
     base congestion-control-algorithm;
     description
       "CUBIC congestion control";
-    reference "RFC 8312: CUBIC for Fast Long-Distance Networks";
+    reference
+      "RFC 8312: CUBIC for Fast Long-Distance Networks";
   }
 
-  // Important global TCP stack parameters
-
   grouping tcp-global {
+    description
+      "Important global TCP stack parameters.";
     leaf mss-max {
       type uint16;
       description
         "Sets the max segment size for TCP connections.";
-      reference "RFC 1122: Requirements for Internet Hosts -- Communication
-        Layers";
+      reference
+         "RFC 1122: Requirements for Internet Hosts -- Communication
```

Comment:
The indentation here is off by one. My bad. Could you correct it and the next line.

Reply:
Fixed in PR #25